zebra/zebra_dplane.h

```diff
@@ -1226,9 +1226,6 @@ struct zebra_dplane_ctx *dplane_provider_dequeue_in_ctx(
 int dplane_provider_dequeue_in_list(struct zebra_dplane_provider *prov,
 				    struct dplane_ctx_list_head *listp);
 
-/* Current completed work queue length */
-uint32_t dplane_provider_out_ctx_queue_len(struct zebra_dplane_provider *prov);
```

Comment:
hmm, no I'd rather leave this - looks like there's code that accesses this length "raw", and maybe it'd be better to convert that to using the api?

Reply:
I'll get Donna to fix this.  I missed this when I was looking at the code.